src/reportWebVitals.js

```diff
@@ -1,15 +0,0 @@
-const reportWebVitals = (onPerfEntry) => {
```

Comment:
- [ ] You have done well removing this file. However, you still need to remove these marked files in the `public` folder.

  ![iduw](https://github.com/mosidrum/Maths_Magicians/assets/36542986/a9e09923-635f-4753-beb2-d4c95dff8adb)